### DISCUSSION TITLE
AuthorizationError - Invalid Client - Amazon Vendor

### DISCUSSION THREAD

**Discussion by rene-schwabe** [saleweaver/python-amazon-sp-api#1053] [Q&A]
Hello,
I´d like to access Amazon Vendor and receive CatalogItems.

This is my code:

from sp_api.api import CatalogItems
import const

client_config = dict(
    refresh_token=const.REFRESH_TOKEN,
    lwa_app_id=const.LWA_APP_ID,
    lwa_client_secret=const.CLIENT_SECRET,
    aws_secret_key=const.AWS_SECRET_KEY,
    aws_access_key=const.AWS_ACCESS_KEY,
    role_arn=const.ROLE_ARN,
)
res = CatalogItems(credentials=client_config).get_catalog_item(
    "XXXXX", MarketplaceId="A1PA6795UKMFR9"
)

I receive the error `sp_api.auth.exceptions.AuthorizationError: ('invalid_client', 'Client authentication failed', 401)`

The only thing is that the amazon vendor Client Secret start with "amzn1.oa2-cs.v1"
Is this correct? The documentation says it will look like "b5f7f8..."

**Comment by 0xs3n4** [comment]
I have the same problem...

**Comment by iconberg** [comment]
For me on EU market the client secret starts as yours.
Something with your creds is not ok then, did you try a new token from your sellercentral? The app is already registerd there?

**Comment by 0xs3n4** [comment]
The problem was my lwa_clint_id, it was wrong
